### PR TITLE
Added featured owners to featured scenarios

### LINKED ISSUE #4156: Add and show scenario creators for featured scenarios
Most of our featured scenarios are not created by ourselves. This is currently not stated anywhere in the description of the featured scenarios. The scenario descriptions are added by the scenario creators themselves, and may not provide model users any information about the following:

- Who has created the scenario? (organization or authors)
- For what purpose was the scenario created? (a specific study)
- Who has featured the scenario? (Quintel)

I propose we come up with a way to properly provide this information, the main goal being that we communicate clearly to our users that not all featured scenarios are ours. We should think about:

1. How to present this information?
2. What to include? (I have suggested the three bullets above, but they may change)

I'm assigning @noracato for the actual implementation and myself as main responsible for the content. Notifying @Charlottevm @AlexanderWirtz and @thomas-qah, happy to hear your thoughts.

### PR DESCRIPTION
**Pull request by louispt1**
I've added a migration and some minor changes to list a featured scenario owner alongside the featured scenarios in place of the current 'last updated at' attribute listing.

To note: The migration grabs the 'first' owner of a scenario as the default 'featured_owner'. This will need to be updated for our featured scenarios. For future featured scenarios, there is an input in the form to add the featured_owner.

I haven't done anything about the styling of the featured scenarios form because it's internal, but it would probably be good to pick that up at some point as the form (particularly for editing a featured scenario) is not pleasant to look at.

Closes #4156

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -58,7 +59,7 @@
     t.datetime "updated_at", precision: nil
   end
 
-  create_table "multi_year_chart_saved_scenarios", primary_key: ["multi_year_chart_id", "saved_scenario_id"], charset: "utf8mb4", collation: "utf8mb4_0900_ai_ci", force: :cascade do |t|
+  create_table "multi_year_chart_saved_scenarios", primary_key: ["multi_year_chart_id", "saved_scenario_id"], charset: "utf8mb3", force: :cascade do |t|
```
Comment (noracato) [inline db/schema.rb:62]: Give this a check

**Review by noracato — COMMENTED** [review]
To fix the failing spec you should add a default value for the author to `spec/factories/featured_scenario.rb`.

Somehow I like the word 'author' more than 'featured owner', mostly because the word _featured_ is already in the name of the class and we should always try to be as concise as possible. But I'm also good to leave it like this 🙂

File: db/migrate/20240916133558_featured_scenario_creators.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class FeaturedScenarioCreators < ActiveRecord::Migration[7.1]
+  def up
+    add_column :featured_scenarios, :featured_owner, :string
+    FeaturedScenario.reset_column_information
+    FeaturedScenario.all.each do |fs|
+      ss = fs.saved_scenario
+      if ss.owners.any?
+        fs.update_column(:featured_owner, ss.owners.first.user.name)
```
[the remark this answers is not in the document]
Comment (louispt1) [reply db/migrate/20240916133558_featured_scenario_creators.rb:8]: I'm happy to change it to author, i think that's clear.

**Comment by louispt1** [comment]
@noracato I'm going to pick up these suggested changes alongside some other upgrades to this PR that we discussed in the m&d meeting (like swapping the display order of the featured scenarios so most recent ones come first)

**Comment by louispt1** [comment]
@noracato I fixed the schema encoding issues and renamed to author, as well as changing the date to just be the year for the featured scenarios. Featured scenarios also now appear in order of most recent to least recent. I also updated the syntax according to your suggestions and updated the spec so it passes now!
Let me know if you think any further changes are required.

P.s. I kind of assume we're better off using unicode_ci rather than 0900_ai_ci because it is more compatible with older versions of MySQL? Let me know if you disagree / have thoughts on this.

```
charset: "utf8mb4", collation: "utf8mb4_0900_ai_ci", force: :cascade do |t|
charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
```

**Review by noracato — APPROVED** [review]
Note: Right after the deploy, someone has to manually check the featured scenarios to see what happened!

**Pull request merged**